### PR TITLE
Recover windows installer cmake file.

### LINKED ISSUE #43: [BUG] Windows installer is missing files.
**What is the bug?**
A driver can't run if installed by package created by CI. Dependency files are missing.
image
It should contain zlib, libcurl, vld, aws-cpp-sdk, aws-c-sdk and all dependencies.

**How can one reproduce the bug?**
1. Build installer from `main` or download from GH CI
2. Install
3. Try to use driver

**What is the expected behavior?**
Installer should contain all required files.

**What is your host/environment?**
`main` @ [9b8d366](https://github.com/opensearch-project/sql-odbc/commit/9b8d36604a23db6865e8d86f9b50c3de97c1e238)

**Do you have any screenshots?**
TODO

**Do you have any additional context?**
Maybe caused by #27

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
### Description
Partially revert changes from #27.
New installer content:
image
Tested on Windows 32/64 bit and Mac.
 
### Issues Resolved
Fixes #43 
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by forestmvey — APPROVED** [review]

**Review by matthewryanwells — APPROVED** [review]

File: src/installer/CMakeLists.txt [hunk 1]
```diff
@@ -90,4 +90,18 @@ install(FILES "${PROJECT_ROOT}/THIRD-PARTY" DESTINATION doc COMPONENT "Docs")
 # Install resource files
 install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/Resources/opensearch_sql_odbc.tdc" DESTINATION resources COMPONENT "Resources")
 
+if(WIN32)
+    set(AWS_SDK_BIN_DIR "${PROJECT_ROOT}/src/vcpkg_installed/x${BITNESS}-windows/bin")
+    install(DIRECTORY ${AWS_SDK_BIN_DIR} DESTINATION . COMPONENT "Driver" FILES_MATCHING PATTERN "*.dll")
+
+    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
```
Comment (acarbonetto) [inline src/installer/CMakeLists.txt:97]: is this input parameter by the user?  Should we check case-insensitive?

File: src/installer/CMakeLists.txt [hunk 2]
```diff
@@ -90,4 +90,18 @@ install(FILES "${PROJECT_ROOT}/THIRD-PARTY" DESTINATION doc COMPONENT "Docs")
...
+        # We actually never build the installer for Debug
+        install(FILES "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Debug/libcurl-d.dll" DESTINATION bin COMPONENT "Driver")
```
Comment (acarbonetto) [inline src/installer/CMakeLists.txt:99]: Might be easier to read if you just include CMAKE_BUILD_TYPE as part of the string, like:
```
install(FILES "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/" CMAKE_BUILD_TYPE "/libcurl-d.dll" DESTINATION bin COMPONENT "Driver")
```

**Review by acarbonetto — APPROVED** [review]

File: src/installer/CMakeLists.txt (hunk 1 above)
Comment (Yury-Fridlyand) [reply src/installer/CMakeLists.txt:97]: Maybe. I just recovered deleted piece of file. The only thing I added is ` FILES_MATCHING PATTERN "*.dll"` to exclude big `pdb` files (debug symbols).

File: src/installer/CMakeLists.txt (hunk 2 above)
Comment (Yury-Fridlyand) [reply src/installer/CMakeLists.txt:99]: File name of libcurl changes between release and debug

**Pull request merged**